src/server/pfs/server/server_test.go

```diff
@@ -2984,6 +2984,14 @@ func TestGlob(t *testing.T) {
 	fileInfos, err = c.GlobFile(repo, "master", "dir1/*")
 	require.NoError(t, err)
 	require.Equal(t, numFiles, len(fileInfos))
+	fileInfos, err = c.GlobFile(repo, "master", "/non-existant-glob*")
+	require.NoError(t, err)
+	require.Equal(t, 0, len(fileInfos))
+	fileInfos, err = c.GlobFile(repo, "master", "/non-existant-static-file")
```

Comment:
Same as above.

Reply:
Also, the static file stuff mentioned below.

Reply:
Done.